guides-markdown/multisig-cli.md

```diff
@@ -0,0 +1,463 @@
+Throw a Multisig party for yourself and some friends using simple command line tools.
+```
+
+## Introduction
+
+The goal of this guide is demonstrate a multi-signature collaboration between bcoin
+users who may or may not trust each other. We will use only terminal commands to 
+illustrate the process. For a deeper dive into how multisig transactions are constructed
+in JavaScript with the library, see
+[Creating Multi-signature Transactions](https://bcoin.io/guides/multisig-tx.html).
+All the commands in this guide (and so many others!) are described in the
+[API docs](https://bcoin.io/api-docs). There are important details in those docs
+regarding API keys, wallet tokens, and other security measures that are essential
+for using bcoin on mainnet. For this guide, we will just play in regtest mode and
+skip over those steps.
+
+## Let's throw a party
+
+We're inviting 5 of our best friends: Alice, Bob, Charlie, David, and Erica.
+Each of them will create a multisig wallet on their own computers, and they'll
+share their
+[master extended keys](https://github.com/bitcoin/bips/blob/master/bip-0032.mediawiki#Extended_keys)
+with each other. Once everyone knows everyone else's public key, they will each be
+able to derive multisig receive addresses. After they receive some money, the group be able
+to spend their Bitcoin as long as THREE OF THE FIVE participants agree, and sign.
+
+## Creating wallets
+
+Start by launching `bcoin` in `regtest` mode. We'll use 
+[environment variables](https://github.com/bcoin-org/bcoin/blob/master/docs/configuration.md)
+so we don't have to type the network parameter each time.
+
+```
+$ export BCOIN_NETWORK=regtest
+$ bcoin --daemon
+```
+
+Then we'll create Alice's wallet. There are 
+[many options available](https://bcoin.io/api-docs/#create-a-wallet)
+for wallet creation, but all Alice needs to do is set up her wallet as a 3-of-5.
+We'll also specify Segregated Witness, although that option will be a bcoin default
+[very soon.](https://github.com/bcoin-org/bcoin/pull/827)
+
+```
+$ bwallet-cli mkwallet --id=Alice --witness=true --m=3 --n=5
+```
+
+Right away, if Alice checks her wallet's default account, she'll notice it has not
+been initialized yet. That's because it is waiting for more multisig keys:
+
+```
+bwallet-cli --id=Alice account get default
+
+{
+  "name": "default",
+  "initialized": false,
+  "witness": true,
+  "watchOnly": false,
+  "type": "multisig",
+  "m": 3,
+  "n": 5,
+  "accountIndex": 0,
+  "receiveDepth": 0,
+  "changeDepth": 0,
+  "nestedDepth": 0,
+  "lookahead": 10,
+  "receiveAddress": null,
+  "changeAddress": null,
+  "nestedAddress": null,
+  "accountKey": "tpubDDXZKjYarBCHxq9GHQHfRjRVQm8DuVAu33LoigMfHJri1ynqsWVBQbMWQbFU5nT3vabniRrtgJtaDEqnpB9GKQTyac5SbUw4589SvwrYKqQ",
+  "keys": [],
+  "balance": {
+    "tx": 0,
+    "coin": 0,
+    "unconfirmed": 0,
+    "confirmed": 0
+  }
+}
+```
+
+Notice how she DOES have an `accountKey` (or "xpub" -- on testnet it's a `tpub`).
```

Comment:
`on testnet` - this is technically regtest based on your explanation above, so maybe better to say "on networks other than mainnet"